src/Core/src/Platform/Windows/ButtonExtensions.cs

```diff
@@ -20,5 +29,93 @@ public static class ButtonExtensions
 
 		public static void UpdateFont(this MauiButton nativeButton, IButton button, IFontManager fontManager) =>
 			nativeButton.UpdateFont(button.Font, fontManager);
+
+		public static async Task UpdateImageSourceAsync(this MauiButton nativeButton, IButton button, IImageSourceServiceProvider? provider)
```

Comment:
This also has to run with the text change since they use the same Content property.

Reply:
Right, good point. Updated.